Gruntfile.js

```diff
@@ -49,7 +49,8 @@ module.exports = function (grunt) {
             options: {
                 vendor: [
                     'node_modules/jquery/dist/jquery.min.js',
-                    'node_modules/angular/angular.min.js'
+                    'node_modules/angular/angular.min.js',
+                    '<%= paths.js %>/dist/*.js',
```

Comment:
I figured it must, but I just couldn't get it to go. After diving into the source of `grunt-contrib-jasmine`, this was the only fix I could see.